subscription/src/main/java/org/killbill/billing/subscription/api/user/DefaultSubscriptionBase.java

```diff
@@ -929,8 +929,10 @@ public int compare(final SubscriptionBaseEvent o1, final SubscriptionBaseEvent o
                                             .getApiEventType().toString()));
                     }
                     break;
-                case EXPIRED:
-                	final ExpiredEvent expiredEvent = (ExpiredEvent) cur; //TODO_1533 - Added temporarily to prevent exception while running tests, actual code will come later
+                case EXPIRED://TODO_1533 - See if anything else needs to be done here
```

Comment:
Looks good to me: I think this is correct, you can remove the TODO_1533.

Reply:
Fixed